tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,27 @@
+# For background on this read the following two links
+# https://wiki.gentoo.org/wiki/Power_management/Processor
+# https://wiki.archlinux.org/index.php/CPU_frequency_scaling
+
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled"
+    run_cmd('modprobe acpi-cpufreq')
+  end
+
+  run do
```

Comment:
Maybe make sure the CPU speed services are disabled before doing this, just in case